packages/browser/src/integrations/helpers.ts

```diff
@@ -175,27 +176,43 @@ export function breadcrumbEventHandler(eventName: string): (event: Event) => voi
 
     lastCapturedEvent = event;
 
-    // try/catch both:
-    // - accessing event.target (see getsentry/raven-js#838, #768)
-    // - `htmlTreeAsString` because it's complex, and just accessing the DOM incorrectly
-    //   can throw an exception in some circumstances.
-    let target;
-    try {
-      target = _htmlTreeAsString(event.target as Node);
-    } catch (e) {
-      target = '<unknown>';
+    const captureBreadcrumb = () => {
+      // try/catch both:
+      // - accessing event.target (see getsentry/raven-js#838, #768)
+      // - `htmlTreeAsString` because it's complex, and just accessing the DOM incorrectly
+      //   can throw an exception in some circumstances.
+      let target;
+      try {
+        target = event.target ? _htmlTreeAsString(event.target as Node) : _htmlTreeAsString((event as unknown) as Node);
```

Comment:
`_htmlTreeAsString(event.target || event)` should work as well, but im fine with either.